src/CampaignsAdmin/edit/CampaignDetails.tsx

```diff
@@ -1,15 +1,17 @@
-import React from 'react';
-import { useCampaignById } from '../../db/hooks/getCampaignDetailsByID';
-import { useParams } from 'react-router-dom';
-import { CampaignLocationInfo } from '../../models/CampaignLocationInfo';
-import { Table, Divider, Image, message, Popconfirm, Button } from 'antd';
-import { DueDateLabel } from '../DueDateLabel';
-import { DeleteOutlined } from '@ant-design/icons';
 import { v4 as uuidv4 } from 'uuid';
-import { supabase as supabaseClient } from '../../db/helper';
+import { DeleteOutlined } from '@ant-design/icons';
```

Comment:
We should have a convention for ordering imports. In this example I did: 
- External libraries (antd, react-router, etc)
- Hooks, utilities
- Models and DB

It can be anything you want, and it can be programmed to reorder automatically by a linter, you just have to decide what you want it to be